lib/lz4.h

```diff
@@ -102,8 +102,8 @@ extern "C" {
 #define LZ4_EXPAND_AND_QUOTE(str) LZ4_QUOTE(str)
 #define LZ4_VERSION_STRING LZ4_EXPAND_AND_QUOTE(LZ4_LIB_VERSION)
 
-LZ4LIB_API int LZ4_versionNumber (void);  /**< library version number; useful to check dll version */
-LZ4LIB_API const char* LZ4_versionString (void);   /**< library version string; unseful to check dll version */
+LZ4LIB_API int LZ4_versionNumber (void);  /**< library version number; to be used when checking dll version */
```

Comment:
was `lz4.h` updated from `dev` ?

Some of these comments look like older version comments.

Reply:
Yeah, it looks like there was a rebase problem. I'll fix it.